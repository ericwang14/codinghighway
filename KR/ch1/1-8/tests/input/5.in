text	split	with	a	few	tabs
and some  hiden spaces       