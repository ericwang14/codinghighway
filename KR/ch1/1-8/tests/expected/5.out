Blanks: 11
Tabs: 5
Newlines: 1
